auv_sim/auv_sim_bringup/launch/start_gazebo.launch

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0"?>
 <launch>
+  <env name="GAZEBO_MODEL_PATH" value="$(find auv_sim_description)/models:${GAZEBO_MODEL_PATH}"/>
```

Comment:
Set gazebo model path automatically for everyone for convenience. Works correctly 👍